src/main/java/minion/data/task/Deadline.java

```diff
@@ -0,0 +1,70 @@
+package minion.data.task;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    protected String datetime;
```

Comment:
Perhaps you could write in camelCase?
```suggestion
    protected String dateTime;
```